pygmt/datasets/samples.py

```diff
@@ -285,39 +285,18 @@ def load_fractures_compilation(**kwargs):
     return data[["length", "azimuth"]]
 
 
-def load_hotspots(**kwargs):
+def _load_hotspots():
     """
-    (Deprecated) Load a table with the locations, names, and suggested symbol
-    sizes of hotspots.
-
-    .. warning:: Deprecated since v0.6.0. This function has been replaced with
-       ``load_sample_data(name="hotspots")`` and will be removed in
-       v0.9.0.
-
-    This is the ``@hotspots.txt`` dataset used in the GMT tutorials, with data
-    from Mueller, Royer, and Lawver, 1993, Geology, vol. 21, pp. 275-278. The
-    main 5 hotspots used by Doubrovine et al. [2012] have symbol sizes twice
-    the size of all other hotspots.
-
-    The data are downloaded to a cache directory (usually ``~/.gmt/cache``) the
-    first time you invoke this function. Afterwards, it will load the data from
-    the cache. So you'll need an internet connection the first time around.
+    Load a table with the locations, names, and suggested symbol sizes of
+    hotspots as a pandas.DataFrame.
 
     Returns
     -------
     data : pandas.DataFrame
         The data table with columns "longitude", "latitude", "symbol_size", and
```

Comment:
Thanks for updating `placename` to `place_name`. Just to be consistent with the other datasets:
```suggestion
    data : pandas.DataFrame
        The data table. The column names are "longitude", "latitude",
        "symbol_size", and "palce_name".
```